src/EditorFeatures/VisualBasicTest/ImplementInterface/ImplementInterfaceTests.vb

```diff
@@ -864,22 +864,12 @@ End Class")
         <WorkItem(541092, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/541092")>
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)>
         Public Async Function TestShowForNonImplementedPrivateInterfaceMethod() As Task
```

Comment:
Change the name of this, and probably remove the workitem since it's no longer applicable to this test.